make_queue/templates/make_queue/course/registration_edit.html

```diff
@@ -73,17 +82,24 @@ <h1>
     </div>
 
     <script>
-        $('.message .close')
-            .on('click', function () {
-                $(this)
-                    .closest('.message')
-                    .transition('fade')
-                ;
-            })
-        ;
-        $(".clear-dropdown").click(function() {
+        $('#messages').click(function () {
+            $(this).fadeOut();
```

Comment:
This change away from `$('.message .close')...` means that the message for successful adds will not be closable. So you could probably just revert this change and everything (with the adjustments from the other comment) should work just fine.

Reply:
I completely forot about the confirmation message. Have fixed it now.